hardware/intelfocl/src/vta.cl

```diff
@@ -0,0 +1,341 @@
+#pragma OPENCL EXTENSION cl_intel_channels: enable
```

Comment:
Just curious, as I remember this PR is going to work for ultra96, does this extension work for Xilinx toolchain to compile the hardware design? I might misunderstood somewhere.

Reply:
The code that work for ultra96 is still inside hardware/xilinx